src/AsmResolver.Workspaces.DotNet/Analyzers/Definition/TypeSpecificationAnalyzer.cs

```diff
@@ -21,6 +21,13 @@ protected override void Analyze(AnalysisContext context, TypeSpecification subje
             {
                 context.ScheduleForAnalysis(subject.DeclaringType);
             }
+
+            if (subject.Resolve() is { } definition)
```

Comment:
Why are we resolving here? Doesn't this flatten the dependency graph by skipping any type references?